rosette/h/Ob.h

```diff
@@ -52,9 +52,16 @@
 #endif
 #endif
 
+#ifdef __GNUG__
```

Comment:
I can see that someone was using this before, but what are you gonna do about clang? Just ignore it? I don't like this very much.

Besides, std::uint8_t is a standard type for C++ compilers since C++11. What standard did you pick after the last code review?

http://en.cppreference.com/w/cpp/types/integer

Reply:
I thought we had decided that maintaining support of old and diverse versions of *nix wasn't necessary. Am I missing something?

Reply:
1) Clang is a better C++ compiler than g++
2) I already told you to test __cplusplus and similar standard macros for C++ standard version, instead of checking for a specific tool. I suggest C++11, but C++14 is also reasonable. Both GCC and Clang will tell you the C++ standard they are supporting and this will ensure that the std namespace has what you need. E.g., std::uint8_t.
3) It's fine to target a recent Ubuntu, but let's not make portability more difficult in the process. E.g., OSX doesn't ship with GCC anymore, it ships with LLVM/clang. If you do it my way, then you may get native OSX support nearly "for free" in a month or two.

Reply:
I just did a trial build with clang.  This resulted in *MANY* more warnings than g++ was producing. This isn't necessarily a bad thing, but will require a lot more work to calm down.